skimage/metrics/tests/test_structural_similarity.py

```diff
@@ -239,3 +240,6 @@ def test_invalid_input():
         structural_similarity(X, X, K2=-0.1)
     with testing.raises(ValueError):
         structural_similarity(X, X, sigma=-1.0)
+    # image is too small
```

Comment:
Can you also add a test for when win_size exceeds image extent? Thanks!

Reply:
@mkcor  Thanks for the review!
I think it was already added here!  https://github.com/scikit-image/scikit-image/blob/e9b086d32f6cc5271fd3c565a2a502fa74f93bd4/skimage/metrics/tests/test_structural_similarity.py#L233-L235 
If I am misunderstanding something, please let me know!

Reply:
Oh, right! My intention would be to distinguish between the two cases... maybe:
```py
    # win_size exceeds image extent                                            
    with testing.raises(ValueError) as cm:                                      
        structural_similarity(X, X, win_size=X.shape[0] + 1)
        assert 'win_size exceeds image extent' in cm.exception.msg
```
and
```py
    # image is too small                                                       
    with testing.raises(ValueError) as cm:                                     
        structural_similarity(Z, Z)                                            
        assert 'Image is too small' in cm.exception.msg
```
?

Reply:
I see! Thanks for kindly letting me know. I just pushed new commits regarding these changes.